include/pcl_conversions/pcl_conversions.h

```diff
@@ -107,31 +107,36 @@ namespace pcl_conversions {
   /** PCLHeader <=> Header **/
 
   inline
-  void fromPCL(const pcl::PCLHeader &pcl_header, std_msgs::Header &header)
+  void fromPCL(const pcl::PCLHeader &pcl_header, std_msgs::msg::Header &header)
   {
     fromPCL(pcl_header.stamp, header.stamp);
-    header.seq = pcl_header.seq;
+    // TODO(clalancette): Seq doesn't exist in the ROS2 header
+    // anymore.  Do we need to do anything here?
+    //header.seq = pcl_header.seq;
```

Comment:
@ros2/team should we just get rid of any reference to the sequence number field of the header message here (and below)?